scripts/validate-eks-ci-role-policy/main_test.go

```
@@ -0,0 +1,315 @@
+	"testing"
+
+	"gopkg.in/yaml.v3"
+)
+
+func repositoryInputs(t *testing.T) ([]byte, []byte, []byte) {
+	t.Helper()
+
+	repoRoot := filepath.Join("..", "..")
+	read := func(path string) []byte {
+		t.Helper()
+		contents, err := os.ReadFile(filepath.Join(repoRoot, path))
```

golangci-lint (2.12.2)

[medium] G304: Potential file inclusion via variable

(gosec)